### DISCUSSION TITLE
LeaderBoardAPI 질문입니다.

### DISCUSSION THREAD

**Discussion by Sonak0930** [naverz/zepeto-studio-kor#1086] [5. Q&A]
![image](https://user-images.githubusercontent.com/59792911/181171941-e88e732e-f6ac-4c3f-ba4e-7e378e94f1b4.png)
PrevRanking 이라는 parameter가 있는데 true와 false의 차이를 잘 모르겠습니다.
어떻게 쓰시는 지 아시는 분 있나요?

++ LeaderBoardAPI 정의를 보니 interface처럼 뼈대만 있던데, 실제로 어떻게 쓰는지 보는 법이 있을까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 리더보드에서 초기화 시즌 날짜를 설정하셨다면
이전 시즌 데이터를 보관하고 있기에
이전 시즌 데이터를 가져올지 말지를 true, false로 세팅할 수 있습니다.

가져오고 싶다면 true로 설정해서 사용하여 주세요~
